test/tasks/reset_task_test.rb

```diff
@@ -0,0 +1,124 @@
+require_relative './task_test_base'
+
+module Mongoid
+  class ResetTaskTest < TaskTestBase
+    def test_database_migrate_reset
+      Mongoid::Migrator.migrations_path = [MIGRATIONS_ROOT + "/valid"]
+      invoke("db:migrate")
+      assert_output(<<-EOF
+
+database: mongoid_test
+
+ Status   Migration ID    Migration Name
+--------------------------------------------------
+   up     20100513054656  AddBaselineSurveySchema
+   up     20100513055502  AddSecondSurveySchema
+   up     20100513063902  AddImprovementPlanSurveySchema
+EOF
+      ) { invoke("db:migrate:status") }
+      invoke("db:migrate:reset")
```

Comment:
Same here, I think it's better to have it `down` (or at least partially `down`) to begin with in order to be sure that something was done.